### DISCUSSION TITLE
Ayuda, hace unos días Chamilo me arroja el siguiente mensaje "Lo sentimos, no le está permitido acceder a esta página o su conexión ha caducado. Para volver a la página anterior pulse el enlace inferior o haga clic en el botón "Atrás" de su navegador"

### DISCUSSION THREAD

**Discussion by KevinUrquijo08** [chamilo/chamilo-lms#5773] [General]
Hace unos días ingresé a mi curso de Chamilo, y me encontré con que todos los cursos que había creado ya no me aparecian ni a mi, ni a mis estudiantes, me remití al buzón en donde me llegaban los mensajes de que mis estudiantes habían respondido los ejercicios, y al momento de tratar de dar un feedback o tratar de llegar de alguna manera diferente al curso me arroja el mensaje "Lo sentimos, no le está permitido acceder a esta página o su conexión ha caducado. Para volver a la página anterior pulse el enlace inferior o haga clic en el botón "Atrás" de su navegador".
¿Saben que puedo hacer respecto a esto?

**Comment by KevinUrquijo08** [comment]
Adicionalmente a esto, no se me permite crear nuevos cursos.
Actualmente llevo el control de todo esto desde la app web.

**Reply by chatonito** [reply]
no sé si sea tu caso, cuando aparece ese mensaje a nivel de administrador, puede corresponder a lo siguiente:
chamilo "apunta" cada curso a una carpeta situada en /app/courses en el servidor y si esta no existe, arroja ese mensaje.
si tiene acceso al cpanel, revise si este directorio esta presente. en caso de no estar, puede ser la causa. habrá que ver si no fue borrado. 
espero haber ayudado
atte
dr, agustin rico guzman webmaster Enseñanza Médica CAR
